NK-Ugwu.py

```diff
@@ -0,0 +1,3 @@
+print('Hello World, this is Nk Ugwu with HNGi7 ID HNG-02568 using python for stage 2 task')

```

Comment:
Delete this one

Reply:
i dont understand. this is the update with the email. have you merged one aleady